### PR TITLE
feat: migrated latest fixes

### PR DESCRIPTION
**Pull request by adubovik**
* Added integration tests for max_tokens and stop sequence
* Use number of bytes as token count estimator for AI21 and AWS Titan
* Allow empty messages in each language model
* Supported history truncation via max_prompt_tokens/discarded_messages parameters
* Bumped version of aidial SDK
* Removed 'Assistant' prefix occasionally generated by Titan
* Supported streaming for Titan and Claude

### CODE REVIEW

**Comment by adubovik** [comment]
/deploy-review
>Deployment status: failed

**Comment by adubovik** [comment]
/deploy-review
>Deployment status: failed

File: aidial_adapter_bedrock/llm/model/ai21.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    else:
+        # The default for max tokens is 16, which is too small for most use cases.
+        # Choosing reasonable default.
+        model_kwargs["maxTokens"] = DEFAULT_MAX_TOKENS_AI21
+
+    if model_params.temperature is not None:
+        model_kwargs["temperature"] = model_params.temperature
+    else:
+        # The default AI21 temperature is 0.7.
+        # The default OpenAI temperature is 1.0.
```
Comment (Oleksii-Klimov) [inline aidial_adapter_bedrock/llm/model/ai21.py]: I think we need to apply a scale factor. The OpenAI temperature ranges from 0 to 2, whereas in AI21, it ranges from 0 to 1.

**Comment by alexey-ban** [comment]
/deploy-review
>Deployment status: success

**Review by Oleksii-Klimov — DISMISSED** [review]

File: aidial_adapter_bedrock/llm/model/ai21.py (hunk 1 above)
Comment (adubovik) [reply aidial_adapter_bedrock/llm/model/ai21.py]: Indeed. Fixed.

**Review by Oleksii-Klimov — APPROVED** [review]

**Pull request merged**